tests/object.js

```diff
@@ -121,9 +120,8 @@ describe('Object', () => {
       }
     ], {})
 
-    assert.equal(csv, `a,b,c,d,e\n0,,,1,this`)
+    assert.equal(csv, `a,b,c,d,e`+ os.EOL +`0,,,1,this`)
```

Comment:
Note, no real changes related to this pull request. Looking for supporting unit tests and none found